locales/en.json

```diff
@@ -330,7 +331,7 @@
       "zero": "You can currently not send anything to @{username} which means that there is no trust path between you."
     },
     "bodyMaxFlow": "Maximum amount: {amount}",
-    "bodyPaymentNoteInvalid": "You cannot write special symbols like {}<>^ and more than 100 characters.",
+    "bodyPaymentNoteInvalid": "You cannot write special symbols like {}<br>^ and more than 100 characters.",
```

Comment:
I think we should break out this message to TextVerification.noteInvalid instead of have duplicated messages